### PR TITLE
更新README，添加7月14日会议纪要

### PR DESCRIPTION
**Pull request by Mikefotiaoqing**
更新目录，添加中文会议纪要

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 Date | Topic | Agenda  | Notes | Summary |
 |:---|:---|---|---|:---|
+Jul 14 | Governance | [Agenda](https://github.com/carboclan/pm/issues/32) | [en]() [cn](notes/20190717-meeting-governance-cn.md) |  Contract Design & 产品设计 |
```
Comment (0xSSoul) [inline README.md]: Please change  产品设计 to  Product Design

Comment (0xSSoul) [reply README.md]: Agenda's link of July 14 is : https://github.com/carboclan/pm/issues/28

Comment (0xSSoul) [reply README.md]: change  notes/20190717-meeting-governance-cn.md to  notes/20190714-meeting-governance-cn.md

Comment (Mikefotiaoqing) [reply README.md]: 收到，已修改

**Pull request closed without merging**